app/src/components/market/sections/market_list/market_home_container.tsx

```diff
@@ -63,17 +62,20 @@ const TextTitle = styled.div`
 `
 const TextDescription = styled.div`
   margin-top: 8px;
-  color: ${props => props.theme.colors.textColorLighter};
+  color: ${props => props.theme.text2};
   line-height: 19px;
 `
 const Link = styled.a`
-  color: ${props => props.theme.colors.clickable};
+  color: ${props => props.theme.primary2};
   text-decoration: underline;
 
   &:hover {
-    color: ${props => props.theme.buttonSecondary.color};
+    color: ${props => props.theme.primary3};
   }
 `
+const IconWrapper = styled(IconClose as any)`
+  color: ${props => props.theme.text1};
+`
```

Comment:
So when you make it like this styled(IconClose) that is basically the same as IconClose component and you don't need wrapper for it but you can just use the wrapper you created and it shouldn't be named wrapper. Please spend some time going thought our codebase on how we do this pattern we use it a lot throughout the code and you should understand it....

Reply:
Yes, I'm gonna spend the time to look into this because it's the first time I deal with such a technique...